addons/product/models/product.py

```diff
@@ -678,7 +678,7 @@ def _is_variant_possible(self, parent_combination=None):
         :rtype: bool
         """
         self.ensure_one()
-        return self.product_tmpl_id._is_combination_possible(self.product_template_attribute_value_ids, parent_combination=parent_combination)
+        return self.product_tmpl_id._is_combination_possible(self.product_template_attribute_value_ids, parent_combination=parent_combination, ignore_no_variant=True)
```

Comment:
Docstring can be updated (remove note about `no_variant` issue). (also in `_get_possible_variants`)